packages/faker/index.ts

```diff
@@ -0,0 +1,67 @@
+// @ts-nocheck
```

Comment:
Where is this module used? I can't see it imported anywhere.

Do we need a new package just for this one file? Feels like this could just sit inside either `runner-shared`, or `reporter` - will it be used anywhere else?

Reply:
I'm using it in the set of tests I'm writing now.